### PR TITLE
fix: FormItem validateWrapper style

### PR DESCRIPTION
**Pull request by maxmeng93**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
https://github.com/ant-design/ant-design/issues/19741

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | fix: FormItem validateWrapper style |
| 🇨🇳 Chinese |修复 FormItem 验证样式 |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by maxmeng93** [comment]
https://github.com/ant-design/ant-design/issues/19741

File: components/form/FormItem.tsx [hunk 1]
```diff
@@ -257,7 +257,7 @@ export default class FormItem extends React.Component<FormItemProps, any> {
     let classes = `${prefixCls}-item-control`;
     if (validateStatus) {
       classes = classNames(`${prefixCls}-item-control`, {
-        'has-feedback': props.hasFeedback || validateStatus === 'validating',
```
Comment (afc163) [inline components/form/FormItem.tsx]: 目前不能直接删，校验中状态会有 loading 图标要展现。

感谢排查，下午或明天我找个时间再看下。

Comment (maxmeng93) [reply components/form/FormItem.tsx]: ok

Comment (maxmeng93) [reply components/form/FormItem.tsx]: @afc163

**Comment by yoyo837** [comment]
This issue has been fixed in 4.0. #20691

**Pull request closed without merging**